packages/react/src/patterns/blocks/ContentBlockSimple/__stories__/ContentBlockSimple.stories.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import readme from '../README.md';
+import { settings } from 'carbon-components';
+import { storiesOf } from '@storybook/react';
+
+const { prefix } = settings;
+
+storiesOf('Patterns (Blocks)|ContentBlockSimple', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const ctaProps = {
+      style: 'card',
+      type: 'external',
+      title: 'Lorem ipsum dolor sit amet',
+      href: 'https://www.ibm.com',
+      copy: 'Lorem ipsum dolor sit ametttt',
+    };
+
+    const cta = {
+      cta: ctaProps,
+      none: null,
+    };
+
+    const image = {
+      images: [
+        {
+          src: 'https://fpoimg.com/320x180?bg_color=0f62fe&text_color=ffffff',
+          minWidth: 320,
+        },
+        {
+          src: 'https://fpoimg.com/400x225?bg_color=0f62fe&text_color=ffffff',
+          minWidth: 400,
+        },
+        {
+          src: 'https://fpoimg.com/672x378?bg_color=0f62fe&text_color=ffffff',
+          minWidth: 672,
+        },
+      ],
+      alt: 'CTA image',
+      defaultImage:
+        'https://fpoimg.com/672x378?bg_color=0f62fe&text_color=ffffff',
+    };
+
+    const copy = {
+      'single paragraph': `Lorem    ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+      Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
+      nulla quis, consequat libero. Here are
+      some common categories:`,
+      'multiple paragraphs': `   Lorem    ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+      Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
+      nulla quis, consequat libero. Here are
+      some common categories:
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+      `,
+      'multiple paragraphs (styled)': `   Lorem    ipsum *dolor* sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+      Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
+      nulla quis, *consequat* libero. Here are
+      some common categories:
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+      `,
+    };
+
+    return (
+      <div className={`${prefix}--grid`}>
```

Comment:
Shouldn't be a deal breaker, but you're using prefix here, and not below. I don't think prefix is necessary since we're bringing the grid into storybook as is.